travelpal/assets/js/components/Profile.jsx

```diff
@@ -24,39 +24,42 @@ export default function Profile({
     updateSearch(query);
   }
 
-  function toggleDefault() {
-    $("#searchFriends").show();
-    $("#profileFriends").hide();
-    $("#searchBtn").hide();
-    $("#viewBtn").show();
-  }
-  function toggleReverse() {
-    $("#searchFriends").hide();
-    $("#profileFriends").show();
-    $("#searchBtn").show();
-    $("#viewBtn").hide();
+  function myToggle() {
+    $("#searchFriends").toggle();
+    $("#profileFriends").toggle();
+    $("#searchBtn").toggle();
+    $("#viewBtn").toggle();
   }
 
-  const searchBtn = (
-    <Button id={"searchBtn"} type="button" onClick={() => toggleDefault()}>
```

Comment:
this shouldnt be deleted. I believe we should keep this. 
here are my reasons:
asd
asd
asdsadasdasd

Reply:
I think we should switch to Typescript and update our style guides.